p/scripts/main.js

```diff
@@ -1286,7 +1296,57 @@ function init_crypto_form() {
 }
 //</crypto form (Web login)>
 
+var sidebar = document.getElementById('sidebar');
+var useJsScrollbar = true;
+try {
+	useJsScrollbar = sidebar && !CSS.supports('scrollbar-color: auto') &&
+		!(parseInt(getComputedStyle(sidebar, '::-webkit-scrollbar').width) < 99);
```

Comment:
What's this magic number? I generally prefer using a self-explanatory named variable.

Reply:
@Alkarex explained that in a resolved conversation here: https://github.com/FreshRSS/FreshRSS/pull/2117#discussion_r238876288

Reply:
It needs to be a number greater than 1 and smaller than the tree width. I am not sure a constant would make sense. I suggest to add a comment if needs be

Reply:
Alternatively, could be changed by querying the tree width. I think I will do that

Reply:
Addressed in https://github.com/FreshRSS/FreshRSS/pull/2117/commits/a3074b46873369081e37c629d096f7eb82fb935f